Makefile

```diff
@@ -93,6 +100,15 @@ domain:
 	$(eval ENV_SHORT=pd)
 	$(eval ENV_TAG=prod)
 
+.PHONY: aks_domain
+aks_domain:
+	$(eval DEPLOY_ENV=production)
+	$(eval AZURE_SUBSCRIPTION=s189-teacher-services-cloud-production)
+	$(eval RESOURCE_NAME_PREFIX=s189p01)
+	$(eval ENV_SHORT=pd)
+	$(eval ENV_TAG=prod)
+	$(eval RG_TAGS=$(shell echo '{"Portfolio": "Early Years and Schools Group", "Parent Business":"Teacher Training and Qualifications", "Product" : "Teaching Record System", "Service Line": "Teaching Workforce", "Service": "Teacher Training and Qualifications", "Service Offering": "Database of Qualified Teachers", "Environment" : "$(ENV_TAG)"}' | jq . ))
```

Comment:
Can we simply set RG_TAGS as in [the template](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/Makefile#L3) ?
Only "Product" is required